python/paddle/fluid/transpiler/distribute_transpiler.py

```diff
@@ -1393,7 +1393,7 @@ def _get_input_map_from_op(self, varmap, op):
 
     def _get_output_map_from_op(self, varmap, op):
         """Returns a dict from op output name to the vars in varmap."""
-        iomap = dict()
+        iomap = collections.OrderedDict()
```

Comment:
Why we need OrderedDict here?

Reply:
because ```dict``` is unordered in Python3, which will cause non-Idempotent in dispatch